### ISSUE TITLE
Notes and filters broken in PM

### ISSUE THREAD

**Issue by alexa-v2** [Amroslayer/Amroslayeruserbot#1]
This seems to be caused by something to do with get_cahtTitle. Trying to save a note or filter in pm generates a crash log

**Comment by Amroslayer** [comment]
Yes i know it,i will try to fix it, i hope i can fix it, thanks for the issues notice :)
